src/pages/ImageMapProView/ImageMapProView.jsx

```diff
@@ -21,32 +21,68 @@ import {
 } from '../../utils/waitForElementToBeAppended';
 
 import './ImageMapPro.scss';
+import HoursModal from '../../components/HoursModal/HoursModal';
 
 const ImageMapProView = () => {
   const [deskId, setDeskId] = useState('');
   const [floorName, setFloorName] = useState('');
-  const location = useLocation();
-  const navigate = useNavigate();
-
+  const [alreadyReservedHours, setAlreadyReservedHours] = useState('');
```

Comment:
after reserving a desk for the entire day, I can click on it again and open the modal for reserving it